sympy/concrete/summations.py

```diff
@@ -253,6 +258,105 @@ def _eval_simplify(self, ratio, measure):
     def _eval_summation(self, f, x):
         return None
 
+    def is_convergent(self):
+        """
+        Convergence tests are used for checking the convergence of
+        a series. There are various tests employed to check the
+        convergence, returns True if convergent and false if divergent and
+        None if unknow. Like divergence test, root test, integral test,
+        alternating series test, comparison tests.
+
+        References
+        ==========
+
+        .. [1] https://en.wikipedia.org/wiki/Convergence_tests
+
+        Examples
+        ========
+
+        >>> from sympy import Interval, factorial, oo, S, Sum
+        >>> from sympy.abc import n
+        >>> Sum(n/(n - 1), (n, 4, 7)).is_convergent()
+        True
+        >>> Sum(n/(2*n + 1), (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(factorial(n)/5**n, (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent()
+        True
+        """
+        from sympy import singularities, Interval, symbols
+        from sympy.solvers.solveset import solveset
+        p, q = symbols('p q', cls=Wild)
+
+        sequence_term = self.args[0]
+        sym = self.args[1][0]
+        lower_limit = self.args[1][1]
+        upper_limit = self.args[1][2]
+        interval = Interval(lower_limit, upper_limit)
+
+        sing = solveset(sequence_term.as_numer_denom()[1], sym, S.Reals)
+        try:
+            for singul in sing:
+                if singul.is_integer and singul in interval:
+                    return False
+        except TypeError:
+            pass
+
+        if lower_limit.is_finite and upper_limit.is_finite:
+            return True
+
+        # transform sym -> -sym and swap the upper_limit = oo and lower_limit = - upper_limit
+        if lower_limit is -oo:
+            sequence_term = sequence_term.xreplace({sym: -sym})
+            lower_limit = -upper_limit
+            upper_limit = oo
+
+        ###  ------- Divergence test ---------- ###
+        try:
+            if limit(sequence_term, sym, upper_limit) != S.Zero:
+                return False
+        except NotImplementedError:
+            pass
+
+        ### ---------- root test --------------- ###
+        lim = limit(abs(sequence_term)**(1/sym), sym, oo)
+        if lim < 1:
```

Comment:
I do not understand your argument. The [wikipedia article](https://en.wikipedia.org/wiki/Root_test) on root_test states.
The root test states that:
`C = limit(sup(abs(a_n)**(1/n)), n -> oo)`
if `C < 1` then the series converges absolutely,
if `C > 1` then the series diverges,
if `C = 1` and the limit approaches strictly from above then the series diverges,
otherwise the test is inconclusive (the series may diverge, converge absolutely or converge conditionally).